plugins/in_kafka/in_kafka.c

```diff
@@ -180,8 +180,11 @@ static int in_kafka_collect(struct flb_input_instance *ins,
 
         rd_kafka_message_destroy(rkm);
 
-        /* TO-DO: commit the record based on `ret` */
-        rd_kafka_commit(ctx->kafka.rk, NULL, 0);
+        
+        if(!ctx->enable_auto_commit) {
+            /* TO-DO: commit the record based on `ret` */
```

Comment:
I think we need to sort all TODOs, we can't just leave them in there. I know it was in the original but we should attempt to sort or add more info.

Reply:
Well i din't introduce the TO-DO. It existed before. It just went into the if statement. Unfortunately github shows to little context so you can't see it.